process-automation/latest/using/index.md

```diff
@@ -205,6 +205,21 @@ The columns that are visible can be adjusted.
 
 ![Select columns]({% link process-automation/images/select-columns.png %})
 
+### Column width
+
+You can adjust the width of the columns of the **Process Management** section. To do this select the edge of the columns and adjust them to the desired width. Once you have changed one column, all the columns with the same name are also adjusted.
```

Comment:
You can add that the column size is saved in preferences. If user will log out and in, it should stay the same.

Reply:
Done.